src/view/list-filters-view.js

```diff
@@ -1,41 +1,41 @@
 import AbstractView from '../framework/view/abstract-view.js';
 
-function createTripFiltersTemplate() {
-  return (
+function createTripFiltersTemplate([type, count], isSelected) {
+  return (/*html*/
     `<div class="trip-filters__filter">
-      <input id="filter-everything"
+      <input
+      id="filter-${type}"
```

Comment:
Не забывай форматировать  и все атрибуты ниже аналогично
```suggestion
      <input
        id="filter-${type}"
```